src/vs/workbench/parts/extensions/electron-browser/extensionsList.ts

```diff
@@ -183,15 +183,15 @@ export class Renderer implements IPagedRenderer<IExtension, ITemplateData> {
 
 		data.name.textContent = extension.displayName;
 		data.author.textContent = extension.publisherDisplayName;
-		data.description.textContent = extension.description;
+		data.description.textContent = extension.id.indexOf('language-pack-') === -1 ? extension.description : ''; // reduce flickering due to wait for manifest
```

Comment:
Its not mandated that the extension id for language packs consists of the word "language-pack". Yes, the ones we have currently all do, but the idea of langauge packs as extensions was to allow community members to create ones for all other languages in which case, we cannot mandate the text in the extension id.

Another way to fix this is to not show the description here, but only show it after the the `getManifest` promise is resolved. There would be no flicker, but the descriptions may come noticeably later.

The flicker appears only when searching for language packs which is not a common occurrence.
I would vote for not trying to fix this at all.

Reply:
I tried only showing it after `getManifest`, but it was very noticeably delayed everywhere. Fine with not fixing it